exporters/fluentd/CMakeLists.txt

```diff
@@ -18,7 +18,20 @@ project(opentelemetry-fluentd)
 
 add_definitions(-DHAVE_CONSOLE_LOG)
 add_definitions(-DENABLE_LOGS_PREVIEW)
-find_package(opentelemetry-cpp REQUIRED)
+
+find_package(opentelemetry-cpp QUIET)
+set(opentelemetry_cpp_clone FALSE)
```

Comment:
Is this variable used?

Reply:
No, was using it earlier and then forgot to remove it. Fixed now.